frontend/components/transaction/SwapTransactionTable.vue

```diff
@@ -40,6 +40,15 @@
                 <span v-else class="opacity-50">Unknown</span>
             </div>
         </div>
+        <div class="flex w-full justify-between">
+            <div>Estimated Profitability Time</div>
+            <div>
+                <template v-if="auctionTransaction.isActive && auctionTransaction.transactionProfitDate">
```

Comment:
Why `v-if` couldn't be the part of the original component? I think it would make sense to encapsulate all related logic in one place